### ISSUE TITLE
[Feature]: multible faces

### ISSUE THREAD

**Issue by kalle07** [Gourieff/sd-webui-reactor#445]
### Feature description

you allready have an option to set which face should be replaced
(Source Image (above):
Comma separated face number(s); Example: 0,2,1)

maybe you can add that i can choose 2 ore more faces models, so i can swap two faces with different face models  ;)

**Comment by kalle07** [comment]
and iam not shure, if i have 2 faces in the image sometimes seems the weight of the face is divided on the two ?!??
maybe cause it if i write some numbers in
Source Image (above):

can you explane that pls more ?

**Comment by ertan2002** [comment]
Did you solve this issue?

**Comment by kalle07** [comment]
no...
I need an explanation for that part in main tab

"Source Image (above):"

**Comment by ertan2002** [comment]
> no... I need an explanation for that part in main tab
> 
> "Source Image (above):"

I found that.

You have to put one Image that contains two people side by side. And give the numbers like 0,1 
0 is the first person and 1 is the second person on the image.

**Comment by kalle07** [comment]
yes but you have two input fields (what that mean if you write in first 0,1 and in sec 1,0 ; only as an example
you can write 0,1 -> 1,1 and lot of combinations 
and 
what is first face most top most left most right ? ^^
lot more combinations,  hope the prgrammer is reading that  ;)

**Comment by ertan2002** [comment]
> yes but you have two input fields (what that mean if you write in first 0,1 and in sec 1,0 ; only as an example you can write 0,1 -> 1,1 and lot of combinations and what is first face most top most left most right ? ^^ lot more combinations, hope the prgrammer is reading that ;)

I don't know why you are thinking in that way. It s very easy.

Ok now you have a single image like 
![Screenshot 2024-07-20 at 01 48 06](https://github.com/user-attachments/assets/27e070ce-daca-4b09-b571-2461523c60f0)
the guy is 0 and the woman is 1 ok.
And I have a video/image that there are one male and one female. If the male on the video on the left side, I write 0,1 that means. from the input image, take the male (0) and swap the male face (0) on the video. And same for woman 1 to 1

Let say, you have only one woman on image but two people on the video and you want to put her face both people on the video.
your input must be 0
and target image must be 0,1
then it will be like this

![image](https://github.com/user-attachments/assets/31deb3ec-8929-41df-8dfc-1d731f92b2d7)

**Comment by kalle07** [comment]
every time 0 is boy and 1 is girl?

ok my case
i have 2girls and i only want replace sometimes the left girl or the right girl... 
how the face-recognition works? which girl is the first 0 the more left the more top?

**Comment by ertan2002** [comment]
> every time 0 is boy and 1 is girl?
> 
> ok my case i have 2girls and i only want replace sometimes the left girl or the right girl... how the face-recognition works? which girl is the first 0 the more left the more top?

It is not about the gender. I just gave you an example.

0 is the first person on the image (that is left side)
1 is the second person on the image (that is right side)

if on your image there are 2 girls, and if you want to swap the face 1-1 (0 to 0 and 1 to 1)
then in both side you have to write 0,1 
that's all.

The example I added, I put one black and one white person. So you can then understand which face is swapped. So for your example, choose very different color skin and play with the numbers. Then you will find the answer

**Comment by kalle07** [comment]
so its safe that the faces counted from left to right not top to down...
ok i will try
